### PR TITLE
Translate "and" and show correct localized chapter name

### PR DESCRIPTION
**Pull request by bazzadp**
Cleans up a few places where we were still using English.

@ksakae1216 has Google Translate chosen the right Japanese word for "and" here or is there a better one?

```
{% block and %}そして{% endblock %}
```

It appears when we have multiple authors:

image

Currently that's incorrectly showing as "and":

image

### CODE REVIEW

**Comment by ksakae1216** [comment]
@bazzadp This is no good, And this has 2 problems.

The following is correct.
↓↓↓
Una Kravets と Adam Argyle によって書かれた。

File: src/templates/ja/2019/base.html [hunk 1]
```diff
@@ -41,6 +41,8 @@
 {% block open_the_index %}目次を開く{% endblock %}
 {% block close_the_index %}目次を閉じる{% endblock %}
 
+{% block and %}そして{% endblock %}
```
Comment (ksakae1216) [inline src/templates/ja/2019/base.html:44]: `{% block and %} と {% endblock %}`

Comment (ksakae1216) [reply src/templates/ja/2019/base.html:44]: This is correct.

Comment (bazzadp) [reply src/templates/ja/2019/base.html:44]:
```suggestion
{% block and %} と {% endblock %}
```

**Comment by bazzadp** [comment]
Like this?

image

**Comment by bazzadp** [comment]
Or even this?
image

**Comment by bazzadp** [comment]
OK will go with the second screenshot.

**Pull request merged**